### PR TITLE
Improve logging for livevalidator

### PR DESCRIPTION
**Pull request by raych1**
Added an optional logging callback to consume by livevalidator to dump
logs to external caller.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 10/14/2019 0.19.8
+
+- Add an optional logging callback to consume by livevalidator in order to dump log to external caller.
+
+## 10/12/2019 0.19.7
```
Comment (lirenhe) [inline ChangeLog.md]: I didn't bump the version to 0.19.7 for those 2 change. 
Could you merge them together with 0.19.8

File: lib/validators/liveValidator.ts [hunk 2]
```diff
@@ -158,6 +165,12 @@ export class LiveValidator {
     this.options = ops as LiveValidatorOptions
   }
 
+  private loggingCallback(message: string, level: number = winston.config.npm.levels.info): void {
```
Comment (lirenhe) [inline lib/validators/liveValidator.ts]: Could you use string as level directly?  I don't think oav need to depend on winston

File: lib/validators/liveValidator.ts [hunk 3]
```diff
@@ -656,7 +691,11 @@ export class LiveValidator {
         onlyFiles: true,
         unique: true
       })
-      log.debug(
+      log.info(
```
Comment (lirenhe) [inline lib/validators/liveValidator.ts]: Could you merge log call and loggingCallback call into one private function?

File: ChangeLog.md (hunk 1 above)
Comment (raych1) [reply ChangeLog.md]: Updated.

File: lib/validators/liveValidator.ts (hunk 2 above)
Comment (raych1) [reply lib/validators/liveValidator.ts]: Updated.

File: lib/validators/liveValidator.ts (hunk 3 above)
Comment (raych1) [reply lib/validators/liveValidator.ts]: Resolved.

**Pull request merged**